sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/directconnectivity/StoreReader.java

```diff
@@ -557,7 +561,11 @@ private ReadReplicaResult createReadReplicaResult(List<StoreResult> responseResu
         });
 
         return storeResultObs.map(storeResult -> {
-            BridgeInternal.recordResponse(entity.requestContext.cosmosResponseDiagnostics, entity, storeResult);
+            try {
+                BridgeInternal.recordResponse(entity.requestContext.cosmosResponseDiagnostics, entity, storeResult);
+            } catch (Exception e) {
+                logger.error("Error occurred while recording response", e);
```

Comment:
Please similar to the original PR, change to "Unexpected failure while recording response"

Reply:
Fixed